src/documents/ComponentConfigurationOptions.md

```diff
@@ -382,8 +382,9 @@ See also: [Component Grades](ComponentGrades.md)
         source: "volumeAsPercent",
         target: "volumeAsFraction",
         singleTransform: {
-            type: "fluid.linearScale",
-            factor: 0.01
+            type: "fluid.transforms.linearScale",
+            factor: 0.01,
+            inputPath: ""
```

Comment:
This can't be right - we don't use the "xxxPath" form in relay documents. This can be removed again